packages/babel-plugin-syntax-nullish-coalescing-operator/src/index.js

```diff
@@ -0,0 +1,7 @@
+export default function() {
```

Comment:
should this be an arrow, since it doesn't need `this`?

Reply:
`function` is ok, since we use it in every other plugin.